clients/src/main/java/org/apache/kafka/clients/tools/VerifiableProducer.java

```diff
@@ -150,8 +174,24 @@ public static VerifiableProducer createFromArgs(String[] args) {
             producerProps.put(ProducerConfig.ACKS_CONFIG, Integer.toString(res.getInt("acks")));
             // No producer retries
             producerProps.put("retries", "0");
```

Comment:
Unrelated but I just saw this -- why isn't this just ProducerConfig.RETRIES_CONFIG?